context.go

```diff
@@ -166,10 +167,22 @@ func (c *Context) Load(key string) (interface{}, error) {
 	return ret, err
 }
 
-// GetDbAndBucket returns the DB handler and the bucket name of the service.
-// The server should have created the database before calling this function.
-func (c *Context) GetDbAndBucket() (*bolt.DB, string) {
-	return c.manager.db, c.bucketName
+// GetAdditionalBucket makes sure that a bucket with the given name
+// exists, by eventually creating it, and returns the created bucket name,
+// which is the servicename + the given name.
+func (c *Context) GetAdditionalBucket(name string) (*bolt.DB, string) {
+	fullName := c.bucketName + name
```

Comment:
what happens if name is empty? I think maybe we need to at least log something as a warning.